src/ATen/templates/TensorDerived.cpp

```diff
@@ -30,7 +35,7 @@ int64_t ${Tensor}::dim() const {
   // See Note [Undefined-dim versus 0-dim]
```

Comment:
Update this code comment because we no longer have `kUndefinedDimensions`

Reply:
good catch (we actually need to update the title of the note, because it's still called that).